### DISCUSSION TITLE
Only for next scan (one-time full scan) Enabled in GUI but disabled in config

### DISCUSSION THREAD

**Discussion by masterwishx** [kagemomiji/airsonic-advanced#242] [Q&A] [answered]
Using docker in Unraid latest version

After disabled only next scan its apper again but disabled in config , Not sure if it is the BUG :

![image](https://github.com/kagemomiji/airsonic-advanced/assets/28630321/2e7cf740-fccb-4401-9c2f-e9801e9b6f55)

**Comment by kagemomiji** [comment]
@masterwishx Thanks for reporting, it's bug.  I have created issue #243.

**Reply by kagemomiji** [reply, marked as answer]
@masterwishx 
Could you try latest edge version? 
 `edge-11.1.3-SNAPSHOT.20230701131000`
I have fixed this bug at this version.

**Reply by masterwishx** [reply]
Thanks, checked, Working Fine
